master/docs/conf.py

```diff
@@ -22,6 +22,17 @@
 
 # -- General configuration -----------------------------------------------------
 
+# Check if [docs] dependencies are installed
+try:
+    import sphinxcontrib.blockdiag
+except ImportError:
+    raise RuntimeError("sphinxcontrib.blockdiag is not installed. Please install documentation dependencies with `pip install buildbot[docs]`") 
+import pkg_resources
+try:
+    pkg_resources.require('docutils>=0.8')
+except resources.ResolutionError:
```

Comment:
Yes, you are right, the error being caught is from the pkg_resources module, and needs to caught as `pkg_resources.ResolutionError` and not `resources.ResolutionError`